blocks/gu_spoverview/block_gu_spoverview.php

```diff
@@ -43,13 +52,90 @@ public function hide_header() {
      *       See myoverview plugin for reference.
      */
     public function get_content() {
-        if ($this->content !== null) {
-          return $this->content;
+        global $USER, $DB, $PAGE, $OUTPUT;
+
+
+        $PAGE->requires->css('/blocks/assessmentglance/styles.css');
+
+        $student_id = $USER->id;
+        $current_yr = date("Y");
+
+        // I took all the assignments whose status was submitted
+        $assigment_submitted    = $DB->get_records_sql("SELECT * FROM `mdl_assign_submission` 
+                                                        WHERE FROM_UNIXTIME(timecreated, '%Y') = $current_yr 
+                                                        AND userid = $student_id AND status='submitted'");
+
+        // I took all assignments for a specific student
+        $assignments            = $DB->get_records_sql("SELECT DISTINCT i.itemname 
+                                                        FROM `mdl_course_modules` cm 
+                                                        INNER JOIN `mdl_modules` m ON m.id = cm.module 
+                                                        INNER JOIN `mdl_grade_items` i ON i.courseid = cm.course
+                                                        INNER JOIN 	`mdl_assign` ass ON ass.name = i.itemname 
+                                                        INNER JOIN `mdl_grade_grades` g ON g.itemid = i.id 
+                                                        INNER JOIN `mdl_user` u ON u.id = g.userid 
+                                                        WHERE m.name IN ('assign') 
+                                                        AND i.itemmodule IS NOT NULL AND u.id = $student_id 
+                                                        AND FROM_UNIXTIME(i.timemodified, '%Y') = $current_yr"); 
+
+        // I took all the assignments that were not yet submit and overdue
```

Comment:
Do not use first person in comments. Sample comment should be: // Query for getting all overdue assignments